src/gui/insEdit.cpp

```diff
@@ -44,6 +44,18 @@ const char* fmParamNames[3][32]={
   {"ALG", "FB", "FMS/PMS", "AMS", "AR", "DR", "D2R", "RR", "SL", "TL", "RS", "MULT", "DT", "DT2", "SSG-EG", "AM", "DAM", "DVB", "EGT", "EGS", "KSL", "SUS", "VIB", "WS", "KSR", "DC", "DM", "EGS", "REV", "Fine", "FMS/PMS2", "AMS2"}
 };
 
+const char* esfmParamLongNames[9]={
+  "OP4 Noise Mode", "Envelope Delay", "Output Level", "Modulation Input Level", "Left Output", "Right Output", "Coarse Tune (semitones)", "Detune", "Fixed Frequency Mode"
+};
+
+const char* esfmParamNames[9]={
+  "OP4 Noise Mode", "Env. Delay", "Output Level", "ModInput", "Left", "Right", "Coarse Tn.", "Detune", "Fixed"
+};
+
+const char* esfmParamShortNames[9]={
+  "RHY", "DL", "OL", "MI", "L", "R", "CT", "DT", "FIX"
```

Comment:
RHY?

Reply:
I forgot to change it to "NOI". Oops. This abbreviated version is unused but I should change it should it get used some day.

Reply:
Changed.